libs/sdk-ui-kit/src/Datepicker/Datepicker.tsx

```diff
@@ -148,20 +152,18 @@ export class WrappedDatePicker extends React.PureComponent<DatePickerProps, IDat
 
     resizeHandler = debounce(() => this.alignDatePicker(), 100);
 
-    private updateDate(date: Date, locale: string) {
-        moment.locale(translationUtils.sanitizeLocaleForMoment(locale));
-
+    private updateDate(date: Date, _locale: string) {
         return this.normalizeDate(date);
```

Comment:
Please check the translation on datepicker. Before it depend on the locale but now it is not.

Reply:
locale is used for `moment` only, and we are not using `moment` anymore